jaas/src/main/java/com/griddynamics/jagger/jaas/service/TestEnvironmentExpirationService.java

```diff
@@ -24,8 +21,11 @@ public TestEnvironmentExpirationService(TestEnvironmentDao testEnvironmentDao) {
 
     @Scheduled(fixedRateString = "${environments.cleaning.job.periodicity}")
     public void deleteExpiredEnvironmentsTask() {
-        long nowTimestampInUtc = now().withZoneSameInstant(UTC).toInstant().toEpochMilli();
-        int deleted = testEnvironmentDao.deleteExpired(nowTimestampInUtc);
-        LOGGER.info(deleted + " expired test environments deleted.");
+        int deleted = testEnvironmentDao.deleteExpired(System.currentTimeMillis());
+        if (deleted > 0) {
+            LOGGER.info("{} expired test environments has been deleted.", deleted);
+        } else {
```

Comment:
Yes. If logs level is DEBUG and `deleted > 0` then there are would be duplicated log